### PR TITLE
Add the changes of better missing data handling in ConjurCA to CHANGELOG

### PR DESCRIPTION
**Pull request by liavyona**
### What does this PR do?
Add the changes of better missing data handling in ConjurCA to CHANGELOG

### What ticket does this PR close?
Connected to #1315

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - A new database migration step updates the fingerprints in slosilo. The FIPS compliance
   update in `v1.8.0` caused the previous fingerprints to be invalid.
   [cyberark/conjur#1584](https://github.com/cyberark/conjur/issues/1584)
+- Verify that Kubernetes Authenticator variables exist and have value before retrieving them so that we raise a
```
Comment (orenbm) [inline CHANGELOG.md:42]: We now verify...

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - A new database migration step updates the fingerprints in slosilo. The FIPS compliance
   update in `v1.8.0` caused the previous fingerprints to be invalid.
   [cyberark/conjur#1584](https://github.com/cyberark/conjur/issues/1584)
+- Conjur now verifies that Kubernetes Authenticator variables exist and have value before retrieving them so that a 
+  proper error will be raised if they aren't and enhance supportability.
+  [cyberark/conjur#1315](https://github.com/cyberark/conjur/issues/1315)
```
Comment (orenbm) [inline CHANGELOG.md:44]: sorry for this but can you remove `and enhance supportability`?

**Review by orenbm — APPROVED** [review]

**Comment by alexkalish** [comment]
Thank you for making this change!  Based on what you wrote, should we have a docs issue for this so that the error is documented?

**Comment by orenbm** [comment]
If the variable is not defined in the policy then a `RequiredResourceMissing` error will be raised. If the variable is defined but has no value then a `RequiredSecretMissing` error will be raised.

**Pull request merged**